### DISCUSSION TITLE
Borders when generating HTML from a spreadsheet

### DISCUSSION THREAD

**Discussion by BruTru** [PHPOffice/PhpSpreadsheet#2571] [Q&A] [answered]
Hello,
With the Xlsx reader, I load a sheet with cells with or without borders.
Then, with the HTML writer, I transform this sheet into an HTML table.
But all the cells of the HTML table are with a border.

I use the example https://phpspreadsheet.readthedocs.io/en/latest/topics/reading-and-writing-to-file/#html which I have adapted

How can I make sure that only the cells with borders in the Excel file are bordered in the HTML file?

Thanks

**Comment by oleibman** [comment, marked as answer]
Do your cells actually have borders, or are they just showing gridlines? If it is the latter, you can eliminate those on any worksheet by:
```php
$spreadsheet->getActiveSheet()->setShowGridlines(false);
```
If that isn't the problem, I would need to see your code to investigate further.

**Reply by BruTru** [reply]
Thank you for the solution. I also found it by analyzing the generated HTML code and searching in Html.php where the gridline class was set.
